keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -311,14 +324,29 @@ the value configured for `trafficDistribution`
 * This leverages existing implementation, requiring no major changes.
 
 #### `PreferClose`
-* **Meaning:** Attempts to route traffic to endpoints within the same zone as
-  the client. If no endpoints are available within the zone, traffic would be
-  routed to other zones.
+* **Meaning:**
+  * Attempts to route traffic to endpoints within the same zone as the client. If no
+    endpoints are available within the zone, traffic would be routed to other zones.
+  * If the Service has endpoints that are part of a DaemonSet deployed to all nodes,
+    then it attempts to route traffic to endpoints on the same node as the client. If
+    no endpoints are available on the same node, traffic would be routed to other
+    nodes (though still preferring the same zone, if relevant).
```

Comment:
Just a comment: This is super great!
Having used iTP of local, its one shortfall is when that endpoint is briefly down (when we upgrade the DaemonSet, for example).
As a user I'd prefer traffic to stay local to the node, but I don't mind if it occasionally leaves the node (or the zone), as long as it's temporary.

This is just a comment, so you're welcome to resolve this.